### PR TITLE
Migrate from @babel/polyfill

### PR DESCRIPTION
**Pull request by Exelord**
Fix: #7 

It uses core-js and runtime-regenerator directly instead of using deprecated `@babel/polyfill` as suggested in the migration guide.

I tested the build files and polyfills on legacy browsers and looks like it works correctly :)

### CODE REVIEW

**Comment by Exelord** [comment]
@pzuraq any feedback?

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,12212 @@
+{
```
Comment (pzuraq) [inline package-lock.json]: This project uses yarn, can you run `yarn` to update the lockfile?

**Comment by Exelord** [comment]
Hmm I didn't test it with babel 6. But I guess it will work in the same way. The difference is only with preset env location and this has not been changed.

Sure If you can, that would be cool to test it.

**Comment by pzuraq** [comment]
Thanks for getting this started!

**Comment by pzuraq** [comment]
Released in v2.0.0

**Comment by Exelord** [comment]
Whoops... I notice some unexpected error after upgrade to v2 :/ 
```
Build Error (broccoli-persistent-filter:TransformAmd) in output/chunk1.js

ember-cli-babel-polyfills: unable to remap: chunk1.js

at TransformAmd.processString (ember-cli-babel-polyfills/lib/transform-amd.js:25:13)

```

**Pull request merged**